examples/cmd/logtransport.go

```diff
@@ -22,15 +21,23 @@ type logTransport struct {
 func (t *logTransport) RoundTrip(r *http.Request) (*http.Response, error) {
```

Comment:
Would it be better if done in the `httpbody.BackupRequestBody` helper?
https://github.com/src-d/metadata-retrieval/pull/57/files#r335450089
That way it will be done for sure always.

Reply:
done by 57eca7e5b